notebooks/Quickstart/quickstart.ipynb

```diff
@@ -0,0 +1,259 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quickstart"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Standard Authentication"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Authenticate with Cromwell using HTTPBasicAuth (username and password)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cromwell_tools.cromwell_auth import CromwellAuth\n",
+    "\n",
+    "auth_httpbasic = CromwellAuth.from_user_password(username='username', \n",
+    "                                                 password='password', \n",
+    "                                                 url='https://cromwell.xxx.broadinstitute.org')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Authenticate with Cromwell using HTTPBasicAuth (secret JSON file)"
```

Comment:
Can you include an example of what this JSON file should look like?

Reply:
Will do!